pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -182,6 +155,106 @@ final class Event {
           'name': name,
         };
 
+  /// Event to capture previous custom dimension values from
+  /// legacy flutter analytics.
+  ///
+  /// The parameters below are a superset of all the implementations
+  /// of the `usageValues` getter in the `FlutterCommand` class within
+  /// the flutter-tool. There should never be a time where all of the parameters
+  /// are passed to this constructor.
+  ///
+  /// The custom dimensions from each implementation are grouped by sharing
+  /// the same prefix. The [workflow] indicates which implementation is being
+  /// sent, for example, "create".
+  Event.commandUsageValues({
```

Comment:
This is a non-obvious name to me (that doesn't mean it isn't the best, just that it's not obvious to my brain). What about just `Event.command`?

Or, do we need to namespace this by the flutter tool somehow? Even though I can see other tools wanting to track its sub-commands, these fields are specific to the flutter tool.

Reply:
I actually had the same thought myself. I'm open to changing the constructor name though, i just tried to make it reflect what it was doing.

But I agree, as we start adding several more events, it's going to be a bit harder to know which event belongs to which dash tool so i created the issue below to at least add the tool it belongs to in the top line in the dartdoc

- https://github.com/dart-lang/tools/issues/212

Reply:
It's your call as to the name